apps/taxes/turbo.json

```diff
@@ -4,23 +4,27 @@
   "pipeline": {
     "build": {
       "env": [
+        "ALLOWED_DOMAIN_PATTERN",
         "APL",
+        "APP_API_BASE_URL",
         "APP_DEBUG",
+        "APP_IFRAME_BASE_URL",
+        "GITHUB_SHA",
+        "NEXT_PUBLIC_SENTRY_DSN",
+        "NEXT_PUBLIC_SENTRY_REPORT_LEVEL",
         "NODE_ENV",
-        "SECRET_KEY",
         "PORT",
-        "VERCEL_URL",
         "REST_APL_ENDPOINT",
         "REST_APL_TOKEN",
-        "ALLOWED_DOMAIN_PATTERN",
-        "SENTRY_ORG",
-        "SENTRY_PROJECT",
-        "SENTRY_DSN",
+        "SECRET_KEY",
         "SENTRY_AUTH_TOKEN",
+        "SENTRY_DSN",
         "SENTRY_ENVIRONMENT",
-        "NEXT_PUBLIC_SENTRY_DSN",
-        "APP_IFRAME_BASE_URL",
-        "APP_API_BASE_URL"
+        "SENTRY_ORG",
+        "SENTRY_PROJECT",
+        "VERCEL_GIT_COMMIT_SHA",
+        "VERCEL_URL",
+        "VERCEL"
```

Comment:
Won't this break caching? VERCEL_GIT_COMMIT_SHA will change on every single commit 🤔

Reply:
All env vars need to be referenced in `turbo.json`. Otherwise, we get an eslint error `turbo/no-undeclared-env-vars`.

Reply:
Could you use this? https://turbo.build/repo/docs/core-concepts/caching/environment-variable-inputs#unhashed-environment-variables

`globalPassThroughEnv` / `passThroughEnv`
This way it won't be included in cache